packages/interpreter/src/interpreters/bear/plugins/noteLinkProcessorPlugin.ts

```diff
@@ -0,0 +1,48 @@
+// @ts-ignore
+import remarkWikiLink from "remark-wiki-link";
+import unified from "unified";
+import unist from "unist";
+import { map as unistUtilMap } from "unist-util-map";
+
+export const noteLinkNodeType = "noteLinkNode";
+export interface NoteLinkNode extends unist.Node {
+  type: typeof noteLinkNodeType;
+  targetNoteName: string;
+  targetNoteDisplayName: string;
+}
+
+function transformWikiLinksIntoNoteLinks(node: unist.Node): unist.Node {
+  return unistUtilMap(node, (visitee) => {
+    if (visitee.type === "wikiLink") {
+      const wikiLinkNode =
+        visitee as unknown as remarkWikiLink.RemarkWikiLinkNode;
+      return {
+        type: "noteLinkNode",
+        targetNoteName: wikiLinkNode.value,
+        targetNoteDisplayName: wikiLinkNode.data.alias,
+      } as NoteLinkNode;
+    } else {
+      return visitee;
+    }
+  });
+}
+
+export default function noteLinkPlugin(this: unified.Processor) {
```

Comment:
I don't think the interpreter actually uses this plugin—may be able to just drop this file altogether.

Reply:
We do, for a simple silly reason: to ignore embedded prompts which are included in the backlinks sections of notes!